web/src/reactions.ts

```diff
@@ -192,40 +233,43 @@ function generate_title(emoji_name, user_ids) {
 }
 
 // Add a tooltip showing who reacted to a message.
-export function get_reaction_title_data(message_id, local_id) {
+export function get_reaction_title_data(message_id: number, local_id: string): string {
     const message = get_message(message_id);
+    assert(message !== undefined);
```

Comment:
This might also be a case where the caller could be responsible for passing us a `message` object and doing this assert itself; lower value than the other one though since the caller didn't start with fetching the message.

Reply:
I'm going to leave this one since it doesn't seem obvious to me that moving the assertion to the caller is better in the way the previous one did.